### PR TITLE
LPS-173349 Set en-US as the default language to fields type text

### PR DESCRIPTION
**Pull request by vngoliveira**
Related issue [LPS-173349](https://issues.liferay.com/browse/LPS-173349)
The problem was half solved with this [pr](https://github.com/liferay-objects/liferay-portal/pull/1664), but another problem was discovered, so I did a new solution.

### CODE REVIEW

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/SearchableContainer.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ const languages = Liferay.Language.available;
 const languageLabels = Object.values(languages).map((language) => {
 	return {label: language};
 });
+const defaultLanguage = Object.values(languageLabels[0]).toString();
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/SearchableContainer.tsx:27]: Hi @vngoliveira, I think you can achieve the same result with  `const defaultLanguage = languageLabels[0].label;` and it's simpler.

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by vngoliveira** [comment]
ci:test:sf

**Comment by vngoliveira** [comment]
ci:test:stable

**Comment by vngoliveira** [comment]
ci:test:relevant

**Comment by vngoliveira** [comment]
ci:test:object

**Comment by naraandrade** [comment]
ci:forward

**Comment by vngoliveira** [comment]
With @naraandrade authorization, I sent this pr manually to Brian Chan [here](https://github.com/brianchandotcom/liferay-portal/pull/130933)

**Pull request closed without merging**